test/nyc-bin.js

```diff
@@ -570,7 +570,7 @@ describe('the nyc cli', function () {
 
   describe('hooks', function () {
     it('provides coverage for requireJS and AMD modules', function (done) {
-      var args = [bin, process.execPath, './index.js']
+      var args = [bin, '--hook-run-in-this-context', process.execPath, './index.js']
```

Comment:
This test still failed for me.  I had to also add the argument `'--hook-require=false'` for this test to pass.